test/integration/api/policies_controller_test.rb

```diff
@@ -55,4 +55,14 @@ def setup
     get edit_admin_service_policies_path(@service)
     assert_response :service_unavailable
   end
+
+  test 'policies edit for members with no permissions' do
```

Comment:
@akostadinov I applied your suggestion for memoization and added tests for the 2 pages mentioned in the JIRA description. They fail on `master` - return a 200 with full details, and pass in this branch.

Reply:
Yeah, indeed, as `accessible_services` is now returning an empty array (after fixing the `permitted_for` scope), fixing this is a side effect.

As mentioned in the PR description, `:policy_registry_ui` doesn't seem to be used, so probably I'll remove it. However, `:policy_registry` might be used somewhere else (maybe the API?), so, I'll keep it as it is.